.changes/next-release/feature-d546e4af-7efd-4c67-aab4-5c3b446305a7.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Respect the default region in config file on first start of the IDE"
```

Comment:
Just to clarify: this means that every time you open the IDE, it will revert to the default region, or only the first time you select the credentials?

Reply:
no - once you select a region it gets persisted.

Reply:
No, only if you have no aws.xml file